### ISSUE TITLE
Voice Control commands are not working on editor

### ISSUE THREAD

**Issue by irfano** [wordpress-mobile/WordPress-iOS#22972]
### Expected behavior
The Voice Control commands related to text, such as text navigation, text selection, text editing, text deletion should work in the captions of the blocks.

### Actual behavior
None of the Voice Control commands related to text, such as text navigation, text selection, text editing, text deletion work in the captions of the blocks.

### Steps to reproduce the behavior
1. Enable Vocie control from the device settings
2. Start the editor.
3. Try to add blocks by using voice commands.
User report: 7859553-zd-a8c

##### Tested on iPad, iOS 17.4.1, Jetpack iOS 24.3

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by brianhurley** [comment]
Step 3 should be create the following blocks: youTube, Twitter, gallery. 

Step 4 should be to add text to the caption of each of the blocks using voice control, and then try to edit the caption using voice control commands related to text editing: text navigation, text editing, text selection, text deletion. Related voice control commands can be found on the accessibility/voice control settings page shown in the image ![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/2214481/c2b22973-363c-4ce9-ac12-cbcd4b4fd977)
